README.md

```diff
@@ -66,26 +66,21 @@ Or in background with:
 docker-compose start
 ```
 
-## Initialize Sulu
+## Create Sulu Project
 
 ```bash
+# Start bash inside of the php container
 docker-compose exec php bash
+
+# Create a new sulu project with composer
 composer create-project "sulu/sulu-minimal:dev-develop" /tmp/project
 cp -RT /tmp/project . && rm -rf /tmp/project/
-bin/adminconsole sulu:build dev --destroy
-```
 
-During the installation, use the following values for initializing th `app/config/parameters.yml` file:
-
-```yml
-parameters:
-    database_driver: pdo_mysql
-    database_host: mysql
-    database_port: null
-    database_name: sulu
-    database_user: user
-    database_password: password
-    database_version: 5.7
+# Set the correct database url to the `.env` file
+sed -i 's/^# DATABASE_URL.*/DATABASE_URL=mysql:\/\/user:password@mysql:3306\/db_name/' .env
```

Comment:
This should be `sed -i 's/^# DATABASE_URL.*/DATABASE_URL=mysql:\/\/user:password@mysql:3306\/sulu/' .env` as `sulu` is the default database name in the `.env` file for docker.

Reply:
Missed that one - thanks! I updated the PR to use the environment variables from the `.env` file to generate the database url.